eng/tsp-core/pipelines/pr-tools.yml

```diff
@@ -64,11 +63,34 @@ extends:
                       --destination-path $(TYPESPEC_WEBSITE_BASE_PATH) \
                       --source "./website/dist/" \
                       --overwrite
+              - task: CopyFiles@2
+                displayName: "Copy VSCode extension .vsix to artifact directory"
+                inputs:
+                  SourceFolder: "$(Build.SourcesDirectory)/packages/typespec-vscode"
+                  Contents: "*.vsix"
+                  TargetFolder: "$(Build.ArtifactStagingDirectory)/vscode-extension"
+            templateContext:
+              outputs:
+                - output: pipelineArtifact
+                  path: $(Build.ArtifactStagingDirectory)/vscode-extension
+                  artifact: vscode-extension
+                  displayName: Copy VSCode extension .vsix to artifact directory
+          - job: tryit_comment
+            displayName: Create TryIt comment
+            dependsOn: tryit
+            steps:
+              - script: |
+                  echo "##[section]Retrieving typespec-vscode artifact URL..."
```

Comment:
we should move this script to a ps1, in this way we could have a more complete script that prints an error if this fails to retrieve the artifact url  or is unable to extra the url

Reply:
updated. @AlitzelMendez

Reply:
I was referring to a file, that instead of having the entire script in the yaml, you could create a file and call it, I feel that the script is quite complex that it belongs to a file, in which you could add more checks and logs if necessary

Reply:
Hello @AlitzelMendez  This script is to get the URL of an pipeline artifact, and it need some pipeline variables include the accessToken which should not allow to be exported out.
I prefer to writing it in the yaml directly.